### PR TITLE
Initial version of Country component.

### PR DESCRIPTION
**Pull request by sebinsua**
Initial version of Country component.

Dependent on: https://github.com/economist-components/component-stats/pull/1

### CODE REVIEW

**Comment by sebinsua** [comment]
I'm going to have a run at approximating the new CSS while I'm waiting for these things to be merged.

File: utils.es6 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function codify(name = '') {
+  return name.toLowerCase().replace(/[^a-z0-9]/g, (chars) => (chars.charCodeAt(0) === 32 ? '-' : ''));
```
Comment (keithamus) [inline utils.es6]: See https://github.com/economist-components/component-stats/pull/1/files#r46059040

> We're already using [`slugify`](https://www.npmjs.com/package/slugify) elsewhere right? Can't we just use it again here?

Comment (sebinsua) [inline utils.es6]: `slugger`. Weirdly I have this memory that it was failing before with an error message about `decodeURIComponent` however now it's working... It's like I'm going crazy.

Will make change here and in the other project.

**Comment by sebinsua** [comment]
@keithamus decode: false!

File: index.es6 [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    </StatsContainer>
+  )
+}
+
+if (process.env.NODE_ENV !== 'production') {
+  const statsContainerPropTypes = {
+    className: PropTypes.string,
+    itemType: PropTypes.string,
+    ...(StatsContainerHeader.propTypes || {}),
+    ...(StatsContainerStats.propTypes || {}),
```
Comment (keithamus) [inline index.es6]: I'm going to merge this, but just an FYI you don't actually need to xor empty object for these.

```js
console.log({
  
  ...undefined,
  ...undefined,
  
})
// => {}
```
[[Babel demo]](https://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=true&playground=false&code=console.log(%7B%0A%20%20%0A%20%20...undefined%2C%0A%20%20...undefined%2C%0A%20%20%0A%7D))

Comment (sebinsua) [inline index.es6]: Good to know. I think I'd noticed that before but for FUD reasons I decided it was a quirk and not a feature...

Comment (keithamus) [inline index.es6]: Nope! https://github.com/sebmarkbage/ecmascript-rest-spread/blob/master/Spread.md
> ##### Null/Undefined Are Ignored
> ```js
let emptyObject = { ...null, ...undefined }; // no runtime error
```

**Pull request merged**